### PR TITLE
PR authors must label their PRs

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

File: .github/workflows/require-semver-labels.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mheap/github-action-required-labels@v1
+        with:
+          mode: exactly
+          count: 1
+          labels: "semver-major, semver-minor, semver-patch, area/ci, area/tests"
```
Comment (jakolehm) [inline .github/workflows/require-semver-labels.yml]: Would syntax `semver/major` match better with other labels (like `area/ci`)?

Comment (Nokel81) [reply .github/workflows/require-semver-labels.yml]: sure, I can change those.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!--
+  All PRs must be labelled with one of the following labels:
```
Comment (chenhunghan) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: What about updating README or other documentation, use "chore"?

Comment (Nokel81) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: Yes that is what that label is for, it will go under "Maintenance" on the change-log.

Comment (Nokel81) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: Other documentation should be put under `area/documentation` I will add that to the set of labels before merging this.

Comment (aleksfront) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: Just reminding about adding `documentation` ^

File: .github/workflows/require-type-labels.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mheap/github-action-required-labels@v1
+        with:
+          mode: exactly
+          count: 1
+          labels: "enhancement, bug, chore, area/ci, area/tests, dependencies"
```
Comment (aleksfront) [inline .github/workflows/require-type-labels.yml]: `documentation` is missing.

**Comment by Nokel81** [comment]
From discussion:

- Might be confusing to require the `semver/*` labels on all PRs

**Review by chenhunghan — APPROVED** [review]
LGTM

**Pull request merged**